<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
  <property name="windowTitle
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QFrame" name="frame_4">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <widget class="QFrame" name="frame_8">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <widget class="QComboBox" name="comboBox">
                    <property name="geometry">
                     <rect>
                      <x>150</x>
                      <y>80</y>
                      <width>391</width>
                      <height>22</height>
                     </rect>
                    </property>
                    <item>
                     <property name="text">
                      <string/>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>TCP</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>UDP</string>
                     </property>
                    </item>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="0">
               <widget class="QFrame" name="frame_9">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="2" column="1">
                  <widget class="QFrame" name="frame_2">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_8">
                    <item row="1" column="0">
                     <widget class="QScrollArea" name="scrollArea">
                      <property name="widgetResizable">
                       <bool>true</bool>
                      </property>
                      <widget class="QWidget" name="scrollAreaWidgetContents_2">
                       <property name="geometry">
                        <rect>
                         <x>0</x>
                         <y>0</y>
                         <width>443</width>
                         <height>216</height>
                        </rect>
                       </property>
                       <layout class="QGridLayout" name="gridLayout_12">
                        <item row="0" column="0">
                         <layout class="QVBoxLayout" name="scroll_layout"/>
                        </item>
                       </layout>
                      </widget>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QScrollArea" name="scrollArea_client">
                      <property name="widgetResizable">
                       <bool>true</bool>
                      </property>
                      <widget class="QWidget" name="scrollAreaWidgetContents_3">
                       <property name="geometry">
                        <rect>
                         <x>0</x>
                         <y>0</y>
                         <width>443</width>
                         <height>216</height>
                        </rect>
                       </property>
                       <layout class="QGridLayout" name="gridLayout_13">
                        <item row="0" column="0">
                         <layout class="QVBoxLayout" name="scroll_layout_2"/>
                        </item>
                       </layout>
                      </widget>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QFrame" name="frame_10">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(252, 249, 246);
border-radius:20px;</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_11">
                    <item row="1" column="0">
                     <widget class="QPushButton" name="activateServer">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton{
	font: 6pt &quot;Ravie&quot;;
	background-color: rgb(255, 214, 153);
	color: rgb(141, 90, 53);
}
QPushButton:hover{
	
	font: 7pt &quot;Ravie&quot;;
	background-color: rgb(238, 198, 143);
}</string>
                      </property>
                      <property name="text">
                       <string>activate Server</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="3">
                     <widget class="QPushButton" name="desactivateClient">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton{
	font: 6pt &quot;Ravie&quot;;
	background-color: rgb(255, 214, 153);
	color: rgb(141, 90, 53);
}
QPushButton:hover{
	
	font: 7pt &quot;Ravie&quot;;
	background-color: rgb(238, 198, 143);
}</string>
                      </property>
                      <property name="text">
                       <string>desactivate Client</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QPushButton" name="activateClient">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton{
	font: 6pt &quot;Ravie&quot;;
	background-color: rgb(255, 214, 153);
	color: rgb(141, 90, 53);
}
QPushButton:hover{
	
	font: 7pt &quot;Ravie&quot;;
	background-color: rgb(238, 198, 143);
}</string>
                      </property>
                      <property name="text">
                       <string>activate Client</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QPushButton" name="desactivateServer">
                      <property name="minimumSize">
                       <size>
                        <width>200</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>200</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton{
	font: 6pt &quot;Ravie&quot;;
	background-color: rgb(255, 214, 153);
	color: rgb(141, 90, 53);
}
QPushButton:hover{
	
	font: 7pt &quot;Ravie&quot;;
	background-color: rgb(238, 198, 143);
}</string>
                      </property>
                      <property name="text">
                       <string>desactivate Server</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QScrollArea" name="scrollArea_system">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(232, 232, 232);</string>
                   </property>
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="scrollAreaWidgetContents_4">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>919</width>
                      <height>98</height>
                     </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_14">
                     <item row="0" column="0">
                      <layout class="QVBoxLayout" name="scroll_layout_3"/>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QFrame" name="frame_5">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(141, 90, 53);
background-color: rgb(252, 249, 246);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="3">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="0">
                  <widget class="QToolButton" name="toolButton">
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Icons/back.png</normaloff>Icons/back.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="2">
                  <widget class="QComboBox" name="socketTypes_2">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>message</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>image</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>video</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>audio</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>pdf</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>word</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>excel</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QComboBox" name="socketTypes">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>TCP</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>UDP</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QFrame" name="frame_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(252, 249, 246);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="2">
                  <widget class="QToolButton" name="sendButton">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Icons/mail.png</normaloff>Icons/mail.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="MessagelineEdit">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="placeholderText">
                    <string>  Enter The Message..</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QToolButton" name="sendFileButton">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>Icons/link.png</normaloff>Icons/link.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>activateServer</sender>
   <signal>clicked()</signal>
   <receiver>desactivateServer</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activateServer</sender>
   <signal>clicked()</signal>
   <receiver>activateServer</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activateClient</sender>
   <signal>clicked()</signal>
   <receiver>desactivateClient</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>944</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activateClient</sender>
   <signal>clicked()</signal>
   <receiver>activateClient</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desactivateServer</sender>
   <signal>clicked()</signal>
   <receiver>desactivateServer</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desactivateClient</sender>
   <signal>clicked()</signal>
   <receiver>desactivateClient</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1000</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>1022</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desactivateClient</sender>
   <signal>clicked()</signal>
   <receiver>activateClient</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>928</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desactivateServer</sender>
   <signal>clicked()</signal>
   <receiver>activateServer</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>


